packages/mdc-linear-progress/mdc-linear-progress.scss

```diff
@@ -15,8 +15,17 @@
 @import "@material/animation/functions";
 @import "@material/theme/mixins";
 @import "./keyframes";
+@import "./mixins";
+@import "./variables";
 
 .mdc-linear-progress {
+  @include mdc-linear-progress-bar-color(primary);
+  @include mdc-linear-progress-buffer-color($mdc-linear-progress-baseline-buffer-color);
+
+  &--accent {
```

Comment:
In the issue description it says to get rid of all --accent modifiers. Also remove reference in README.
Are we looking to remove the concept of primary/accent colours entirely and let users customize to their own colors?

Reply:
Done!

We're still keeping the global `$mdc-theme-primary` and `$mdc-theme-secondary` variables, because they make it easy to create a simple theme with no fancy customizations.

We are, however, removing all the `--accent`/`--secondary` and `--primary` modifier classes from all components - see [my comments on #1332](https://github.com/material-components/material-components-web/issues/1332#issuecomment-335724659) for more background on why we're doing that.